sub-packages/bionemo-esm2/tests/bionemo/esm2/model/test_stop_and_go.py

```diff
@@ -108,3 +113,73 @@ def setup_model(cls, mode: Mode) -> tuple[pl.LightningModule, pl.LightningDataMo
         module = biobert_lightning_module(config=config, tokenizer=cls.tokenizer, optimizer=optimizer)
 
         return module, data, optimizer
+
+
+class TestESM2StopAndGoCheckpointNotAtValidation(TestESM2StopAndGo):
+    @override
+    @classmethod
+    def get_default_callbacks(cls):
+        callbacks = super().get_default_callbacks()
```

Comment:
Do we have both `StopAfterValidEpochEndCallback` and `PreemptionCallback` at the same time here?

Reply:
yup I guess so. We could delete that other one, but I think it's better to test before a validation epoch has occurred so the `StopAfterValidEpochEndCallback` never gets called